src/web/theme/pages/TryItPipe/TryItPipe.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const TryItPipe = () => {
+  return <div>Work In Progress</div>;
+};
+
+TryItPipe.prototype = {};
```

Comment:
It is a WIP so this is not critical but I think you meant "propTypes" here :laughing:

Reply:
X) yep